model/src/channel/message/kind.rs

```diff
@@ -1,6 +1,10 @@
+use crate::channel::ConversionError;
+use core::convert::TryFrom;
```

Comment:
Is there a reason to use `core` instead of `std` here? Other places we are using `std`.

Reply:
Just a habit of mine, I can switch to importing it from `std` if that keeps it lined up better with the rest of the crates.